packages/kbn-apm-synthtrace-client/src/lib/infra/aws/rds.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/* eslint-disable max-classes-per-file */
+import { Entity, Fields } from '../../entity';
+import { Serializable } from '../../serializable';
+
+interface AWSRdsDocument extends Fields {
+  'aws.rds.db_instance.arn': string;
+  'aws.rds.db_instance.identifier': string;
+  'metricset.name'?: string;
+  'event.dataset'?: string;
+}
+
+export class AWSRds extends Entity<AWSRdsDocument> {
+  metrics() {
+    return new AWSRdsMetrics({
+      ...this.fields,
+      'aws.rds.cpu.total.pct': 0.4,
+      'aws.rds.database_connections': 5,
+      'aws.rds.latency.read': 500 * 1000,
+      'aws.rds.latency.write': 500 * 1000,
+      'aws.rds.latency.insert': 500 * 1000,
+      'aws.rds.latency.update': 500 * 1000,
+      'aws.rds.latency.commit': 500 * 1000,
+      'aws.rds.latency.dml': 500 * 1000,
+      'aws.rds.queries': 100,
```

Comment:
I think it's better to move this to the scenario, don't you think? First because we could create different tests, setting different values. Second because if I open the current scenario [created](packages/kbn-apm-synthtrace/src/scenarios/infra_aws_rds.ts) I'm not sure what result to expect, I need to investigate further to find where the values are defined, which in this case is here. Third, if I change any of this values I could possibly break other tests that were expecting the old one.

Reply:
After reviewing this, the `rds.ts` in the client follows a similar pattern as the other files in this `infra/` folder, e.g.: `docker_containers.ts` the `scenarios/infra_docker_containers.ts` do you think the others should be moved to the scenarios as well?

Reply:
I think we could do that later. If you can change the new one I think that is enough for this PR.